### PR TITLE
Expand definition of 'value-range' in equalization docs

### PR DESCRIPTION
**Pull request by Anirban0011**
# What this PR does:

* Updates the definition of value_range argument of Equalization class in preprocessing/layers/equalization.py

### CODE REVIEW

**Comment by Anirban0011** [comment]
#1544

**Comment by Anirban0011** [comment]
#1545

**Comment by Anirban0011** [comment]
#1547

**Comment by Anirban0011** [comment]
@ianstenbit @LukeWood , please have a look at the doc fix.Thanks

File: keras_cv/layers/preprocessing/equalization.py [hunk 1]
```diff
@@ -24,9 +24,9 @@ class Equalization(BaseImageAugmentationLayer):
     """Equalization performs histogram equalization on a channel-wise basis.
 
     Args:
-        value_range: a tuple or a list of two elements. The first value represents
-            the lower bound for values in passed images, the second represents the
-            upper bound. Images passed to the layer should have values within
+        value_range: a tuple, list, numpy array, tf.tensor or any object that can return integer on indexing of two elements.
```
Comment (ianstenbit) [inline keras_cv/layers/preprocessing/equalization.py:27]: While it is true that a numpy array or any int-indexible type can be passed here, we expect users to use a tuple or list, and this makes the docstring a bit harder to parse, so I think we should leave the docs as-is.

**Pull request closed without merging**